Api/ApiConnection.cs

```diff
@@ -109,11 +109,16 @@ public bool TryRequest<T>(RestRequest request, out T result)
                     return false;
                 }
 
+                if (!restsharpResponse.IsSuccessful)
+                {
+                    Log.Error($"ApiConnect.TryRequest(): Content: {restsharpResponse.Content}");
```

Comment:
I'm not sure if we want this here, but it was pretty useful for debugging. How often are we expecting unsuccessful messages, and is there any reason we _shouldn't_ log those responses?

Reply:
This error condition shouldn't be too frequent since exceptions should already be caught on L107,
agree with logging the error here.